FastlaneExample/FastlaneExample.xcodeproj/xcuserdata/berta.xcuserdatad/xcschemes/xcschememanagement.plist

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>SchemeUserState</key>
+	<dict>
+		<key>Beta.xcscheme_^#shared#^_</key>
```

Comment:
Did this actually make it shared? Cos it is still under the `xcuserdata` folder, not the `xcshared` folder. This is the option you need to check:

<img width="953" alt="screen shot 2018-03-19 at 09 41 53" src="https://user-images.githubusercontent.com/666285/37585562-d74f36f2-2b59-11e8-9027-19912eb55615.png">

Reply:
I googled how to make it shared and followed the instructions, did not know about the difference between `xcshared` and `xcuserdata`. Thanks for pointing it out :D

Reply:
Check all the scheme and the shared toggle is applied @amlcurran I am a bit lost tbh if you have 2min today or tomorrow would love some extra 👀

Reply:
My mistake! It has been added it seems, but that class has duplicated it (see [this file](https://github.com/novoda/spikes/pull/373/files#diff-fd71c2db90a87c8f30cee5a69e0c095e)) :+1: